tripy/tripy/frontend/ops/cartesian_via_polar.py

```diff
@@ -0,0 +1,67 @@
+#
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from tripy import export, constraints
+
+
+@export.public_api(document_under="operations/functions")
+@constraints.dtype_info(
+    dtype_variables={"T1": ["float32", "float16", "bfloat16", "float8"]},
+    dtype_constraints={"abs": "T1", "angles": "T1", constraints.RETURN_VALUE: "T1"},
+)
+def cartesian_via_polar(abs: "tripy.Tensor", angles: "tripy.Tensor") -> "tripy.Tensor":
```

Comment:
Why do we need this as an API in Tripy and why can't it be a user function? Can you point to similar functionality provided by torch, jax?

Reply:
This recreates the functionality of [torch.polar](https://pytorch.org/docs/stable/generated/torch.polar.html) and [torch.view_as_real](https://pytorch.org/docs/stable/generated/torch.view_as_real.html). Since we don't support complex numbers, I think it's useful to have an existing function that directly accomplishes the same purpose.

Reply:
Maybe it's better as a helper in the example?